File: ditto-chat-client/src/app/classes/ChatterRegistrationForm.ts
export default class ChatterRegistrationForm {
    private name: string;
    private surname: string;
    private username: string;
    private email: string;
    private password: string;

    public constructor(
        name: string,
        surname: string,
        username: string,
        email: string,
        password: string
    ) {
        this.name = name;
        this.surname = surname;
        this.username = username;
        this.email = email;
        this.password = password;
    }

    public getName(): string {
        return this.name;
    }

    public getSurname(): string {
        return this.surname;
    }

    public getUsername(): string {
        return this.username;
    }

    public getEmail(): string {
        return this.email;
    }

    public getPassword(): string {
        return this.password;
    }
}

File: ditto-chat-server/src/main/java/com/ditto_chat/ditto_chat_server/dtos/ChatterRegistrationForm.java
package com.ditto_chat.ditto_chat_server.dtos;

public class ChatterRegistrationForm {
    private String name;
    private String surname;
    private String username;
    private String email;
    private String password;
    
    public ChatterRegistrationForm(String name, String surname, String username, String email, String password) {
        this.name = name;
        this.surname = surname;
        this.username = username;
        this.email = email;
        this.password = password;
    }

    public String getName() {
        return name;
    }

    public String getSurname() {
        return surname;
    }

    public String getUsername() {
        return username;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }    
}
